Can't resolve dependencies, NestJS 8+

Hi, 

Seems that this package needs to be updated for the 8.x release of NestJs, would be great if it could be updated!

```js
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! Found: @nestjs/common@8.0.4
npm ERR! node_modules/@nestjs/common
npm ERR!   @nestjs/common@"^8.0.4" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer @nestjs/common@"^7.0.0" from @willsoto/nestjs-objection@5.0.0
npm ERR! node_modules/@willsoto/nestjs-objection
npm ERR!   @willsoto/nestjs-objection@"^5.0.0" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
npm ERR! See /root/.npm/eresolve-report.txt for a full report.
```

--- a/package.json
+++ b/package.json
@@ -94,9 +94,9 @@
     "@commitlint/cli": "~12.1.1",
     "@commitlint/config-conventional": "~12.1.1",
     "@istanbuljs/nyc-config-typescript": "~1.0.1",
-    "@nestjs/common": "~7.6.15",
-    "@nestjs/core": "~7.6.15",
-    "@nestjs/testing": "~7.6.15",
+    "@nestjs/common": "~8.0.4",
+    "@nestjs/core": "~8.0.4",
+    "@nestjs/testing": "~8.0.4",
     "@types/chai": "~4.2.17",
     "@types/chai-as-promised": "~7.1.3",
     "@types/mocha": "~8.2.2",
@@ -121,7 +121,7 @@
     "prettier-plugin-packagejson": "~2.2.11",
     "reflect-metadata": "~0.1.13",
     "rimraf": "~3.0.2",
-    "rxjs": "~6.6.6",
+    "rxjs": "~7.2.0",
     "sinon": "~10.0.0",
     "sinon-chai": "~3.7.0",
     "source-map-support": "~0.5.19",
@@ -131,8 +131,8 @@
     "typescript": "~4.3.2"
   },
   "peerDependencies": {
-    "@nestjs/common": "^7.0.0",
-    "@nestjs/core": "^7.0.0",
+    "@nestjs/common": "^7.0.0 || ^8.0.0",
+    "@nestjs/core": "^7.0.0 || ^8.0.0",
     "knex": ">=0.16.0",
     "objection": "^2.0.0"
   },

--- a/src/core.ts
+++ b/src/core.ts
@@ -44,7 +44,7 @@ export class ObjectionCoreModule implements OnApplicationShutdown {
     };
 
     const objectionBaseModelProvider: ValueProvider = {
-      provide: BaseModel.name,
+      provide: BaseModel,
       useValue: BaseModel,
     };
 

--- a/src/module.ts
+++ b/src/module.ts
@@ -28,11 +28,17 @@ export class ObjectionModule {
   }
 
   public static forFeature(models: typeof Model[]): DynamicModule {
-    const modelProviders: Provider[] = models.map((model) => {
-      return {
-        useValue: model,
-        provide: model.name,
-      };
+    const modelProviders: Provider[] = models.flatMap((model) => {
+      return [
+        {
+          useValue: model,
+          provide: model,
+        },
+        {
+          useValue: model,
+          provide: model.name,
+        },
+      ];
     });
 
     return {

--- a/tsconfig.json
+++ b/tsconfig.json
@@ -7,7 +7,7 @@
     "declaration": true,
     "declarationMap": true,
     "forceConsistentCasingInFileNames": true,
-    "lib": ["es2018"],
+    "lib": ["es2019"],
     "module": "commonjs",
     "removeComments": false,
     "sourceMap": true,

